src/meson.build

```diff
@@ -0,0 +1,17 @@
+raylib_src = [
+  'raudio.c',
```

Comment:
cmake has an "audio" option that determines whether to use this.

Reply:
That change was gonna be added after ensuring all including the audio RayLib components compiled.